events/aoe2/aoe.php

```diff
@@ -0,0 +1,899 @@
+
+
+<div id="page" style="background:rgba(16,16,16,0.9); padding-top:20px; padding-bottom:20px; padding-right:20px; padding-left:20px text-color:white; margin-left:auto; margin-right:auto" height="600px" width="800px">
+
+	
+	<p style="color:green;font-size:14px;border:green 2px solid;padding:5px;width:400px;text-align:center;margin-left:auto; margin-right:auto" width="100%">
+        AOE3 Registration Here	</p>
+
+        <center>
+        <span style="color:red;font-size:12px;">
+        </span>
+
+        <form method="POST" action="aoe.php">
+
+        <table cellspacing="5px" style="color:white">
+
+        <tbody><tr>
+
+
+        <tr>
+
+        <td>Clan Name<span style="color: rgb(255, 0, 0); font-weight: bold;">*<span></span></span></td>
+
+        <td><input name="clanname" type="text"></td>
+
+        </tr>
+
+        <tr>
+
+        <td>Member 1<span style="color: rgb(255, 0, 0); font-weight: bold;">*<span></span></span></td>
+        <td><input name="member1" type="text"></td>
+
+        </tr>
+        <tr>
+
+        <td>Member 2<span style="color: rgb(255, 0, 0); font-weight: bold;"><span></span></span></td>
+        <td><input name="member2" type="text"></td>
+
+        </tr>
+        <tr>
+
+        <td>Member 3<span style="color: rgb(255, 0, 0); font-weight: bold;"><span></span></span></td>
+        <td><input name="member3" type="text"></td>
+
+        </tr>
+        <tr>
+
+        <td>Member 4<span style="color: rgb(255, 0, 0); font-weight: bold;"><span></span></span></td>
+        <td><input name="member4" type="text"></td>
+
+        </tr>
+
+        <tr>
+
+        <td>Email<span style="color: rgb(255, 0, 0); font-weight: bold;">*<span></span></span></td>
+
+        <td><input name="email" type="text"></td>
+
+        </tr>
+
+        <tr>
+
+        <td>Country<span style="color: rgb(255, 0, 0); font-weight: bold;">*<span></span></span></td>
+
+
+
+        <td>    <select name="country" size="1" maxlength="10">
+
+        <option> AFGHANISTAN
+
+        </option> <option> ALBANIA
+
+
```

Comment:
This is an interesting spacing. 

I wonder how the creator was automating this :grin: .